### PR TITLE
Implemented the deletion of the incomplete file from download folder

### PR DESCRIPTION
**Pull request by rodrigopavezi**
@stulees @donnapep please review. cheers

### CODE REVIEW

File: app/controllers/file.js [hunk 1]
```diff
@@ -121,6 +122,19 @@ FileController.prototype.moveFileFromDownloadToCache = function() {
   });
 };
 
+FileController.prototype.deleteFileFromDownload = function() {
+  fileSystem.fileExists(this.pathInDownload, (exists) =>{
+    if (exists) {
+      console.log("hey");
```
Comment (stulees) [inline app/controllers/file.js]: @rodrigopavezi Probably forgot to remove this :)

File: app/controllers/file.js [hunk 2]
```diff
@@ -121,6 +122,19 @@ FileController.prototype.moveFileFromDownloadToCache = function() {
...
+      fileSystem.delete(this.pathInDownload, (err) => {
+        if (err) this.emit("delete-file-error", err);
+        console.log("here");
```
Comment (stulees) [inline app/controllers/file.js]: @rodrigopavezi This log too

**Comment by rodrigopavezi** [comment]
@donnapep @stulees opss!!! removed. cheers

**Comment by stulees** [comment]
@rodrigopavezi 👍

**Pull request merged**